ts/sagas/backendInfo.ts

```diff
@@ -21,36 +18,28 @@ const BACKEND_INFO_LOAD_INTERVAL = 60 * 60 * 1000;
 // retry loading backend info every 10 seconds on error
 const BACKEND_INFO_RETRY_INTERVAL = 10 * 1000;
 
-function* backendInfoWatcher(): Generator<
-  Effect,
-  void,
-  SagaCallReturnType<typeof getServerInfo>
-> {
-  const backendPublicClient = BackendPublicClient(apiUrlPrefix);
+function* backendInfoWatcher(): Generator<Effect, void, any> {
+  const contentClient = ContentClient();
 
   function getServerInfo(): Promise<
-    t.Validation<BasicResponseType<ServerInfo>>
+    t.Validation<BasicResponseType<VersionInfo>>
   > {
     return new Promise((resolve, _) =>
-      backendPublicClient
-        .getServerInfo({})
+      contentClient
+        .getVersionInfo()
         .then(resolve, e => resolve(left([{ context: [], value: e }])))
     );
   }
 
   while (true) {
     try {
-      const backendInfoResponse = yield call(getServerInfo);
+      const backendInfoResponse: SagaCallReturnType<typeof getServerInfo> =
```

Comment:
```suggestion
      const versionInfoResponse: SagaCallReturnType<typeof getServerInfo> =
```
I guess there is no 'backend' data in the response

Reply:
I'm working on renaming and refactoring in https://pagopa.atlassian.net/browse/IAI-122, if isn't a problem I would concentrate all the renaming part there